cli/deployments/traefik.go

```diff
@@ -50,12 +50,26 @@ func (k Traefik) Describe() string {
 func (k Traefik) Delete(c *kubernetes.Cluster, ui *ui.UI) error {
 	ui.Note().Msg("Removing Traefik...")
 
+	message := "Deleting Traefik namespace " + TraefikDeploymentID
+	warning, err := helpers.WaitForCommandCompletion(ui, message,
+		func() (string, error) {
+			return c.DeleteNamespaceIfOwned(TraefikDeploymentID)
+		},
+	)
+	if err != nil {
+		return errors.Wrapf(err, "Failed deleting namespace %s", TraefikDeploymentID)
+	}
+	if warning != "" {
+		ui.Exclamation().Msg(warning)
```

Comment:
Same as above

Reply:
Added a comment and exiting only for a specific warning.